Libraries/Components/SafeAreaView/SafeAreaView.js

```diff
@@ -33,7 +35,7 @@ let exported;
 if (Platform.OS === 'android') {
   exported = class SafeAreaView extends React.Component<Props> {
     render(): React.Node {
-      const {emulateUnlessSupported, ...props} = this.props;
+      const {emulateUnlessSupported, onInsetsChange, ...props} = this.props;
```

Comment:
You are destructuring the props here and then not doing anything with `onInsetsChange`. Do you mind patching up the code to make it work properly and making sure the test plan and example show that this is working properly?